pkg/admission/validator/lakom/shoot.go

```diff
@@ -73,9 +141,16 @@ func (s *shoot) Validate(_ context.Context, new, _ client.Object) error {
 	if err := runtime.DecodeInto(s.decoder, lakomExt.ProviderConfig.Raw, lakomConfig); err != nil {
 		return fmt.Errorf("failed to decode providerConfig: %w", err)
 	}
-	if lakomConfig.Scope == nil {
-		return nil
+
+	allErrs := field.ErrorList{}
+
+	if lakomConfig.Scope != nil {
+		allErrs = append(allErrs, s.validateScopeType(providerConfigPath.Child("scope"), *lakomConfig.Scope)...)
+	}
+	if lakomConfig.TrustedKeysResourceName != nil {
+                allErrs = append(allErrs, s.validateTrustedKeys(ctx, providerConfigPath.Child("trustedKeysResourceName"), *lakomConfig.TrustedKeysResourceName, shoot.Spec.Resources, shoot.Namespace)...)
```

Comment:
A bit hard to notice, but here you are using spaces instead of tabs for indentation.